_data/tagging-status.yml

```diff
@@ -8462,13 +8462,12 @@
 
  - name: showlabels
    type: package
-   status: unknown
+   status: compatible
```

Comment:
@FrankMittelbach yes someone using show lablels to edit a document would want them read out so probably not an artifact. I note `showkeys` is marked as partially-compatible with a comment querying the tagging status. In the end they should both end up with similar tagging.  perhaps make tis partially-compatible to match.